src/components/deleteAllChatsButton/DeleteAllChatsButton.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import css from "./styles.module.css";
+
+export function DeleteAllChatsButton(): JSX.Element {
+    const [showConfirmButtons, setShowConfirmButtons] = useState(false);
+    const [isLoading, setIsLoading] = useState(false);
+    const [error, setError] = useState(null);
+    const handleClick = (): void => {
+        setShowConfirmButtons(!showConfirmButtons);
+    };
+
+    const deleteAllChats = () => {
```

Comment:
add jsDoc comment here explaining what it does and why the loading state is used instead of inlining several individual comments